bitnami/elasticsearch/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: elasticsearch
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/elasticsearch
   - https://www.elastic.co/products/elasticsearch
-version: 19.4.6
+version: 19.4.7
```

Comment:
Sorry for not spotting this before. As you are adding new parameters to the chart, this must be a new minor version 

```
19.5.0
```